.github/workflows/lint-test.yml

```diff
@@ -87,9 +89,17 @@ jobs:
     - name: Set up environment
       id: setup-env
       uses: ./.github/actions/setup-env
+      env:
+        PDM_USE_VENV: true
+        PDM_VENV_IN_PROJECT: true
       with:
         python-version: ${{ matrix.python-version }}
-        nox-setup: "*"
+
+    - name: Install dependencies out of venv
+      run: |
+        pdm export --pyproject --without-hashes -G:all -dG:all -o requirements.txt
```

Comment:
`-G:all` also selects the shim, which was previously only specified in `setup.py` (preventing nox from installing it).
This applies to a few places in this PR.

Reply:
I'm not sure how to unselect that, to be honest

Reply:
Ok, I have a solution in the next push.

Reply:
https://github.com/DisnakeDev/disnake/pull/836/commits/0b3a299775314c2049ff9f5f8cb83cd3e5989e2c